byway/src/main/java/com/google/sps/data/Trip.java

```diff
@@ -105,6 +114,7 @@ public Key getKey() {
    * make a valid Trip instance.
    *
    * @param tripEntity entity from datastore
+   * @return Trip object of the Trip class
```

Comment:
This doesn't really tell me anything that's not in the method's signature. Either just leave it out or be more specific, like `@return {@link Trip} with properties copied from the provided {@link Entity}.`

Reply:
I don't think the Cloud Shell editor supports `@link` but I agree, I changed it to give more detail.